framework/doc/content/source/interfaces/Coupleable.md

```diff
@@ -1,31 +1,57 @@
-adCoupledGradient | Gradients of a coupled variable in q-points with automatic differentiation info
-adCoupledSecond | Second derivatives of a coupled variable in q-points with automatic differentiation info
-coupledNodalValue | Values of a coupled variable at nodes.
-adCoupledNodalValue | Values of a coupled (vector) variable at nodes with automatic differentiation info
-coupledVectorValue | Values of a coupled vector variable in q-points
-adCoupledVectorValue | Values of a coupled vector variable in q-points with automatic differentiation info
-coupledCurl | Curl of a coupled vector variable in q-points
-coupledDot | Time derivative of a coupled variable
-coupledDotDu | Derivative of a time derivative of a coupled variable
-coupledNodalDot | Nodal value of the time derivative of a coupled variable
-coupledVectorDot | Time derivative of a coupled vector variable
-coupledVectorDotDu | Derivative of a time derivative of a coupled vector variable
-adCoupledLowerValue | Value a coupled lower-dimensional variable with automatic differentiation info
-
-For values, gradients and second derivatives, users can request old and older values in case they are running a transient simulation.
-In case of old and older values, the methods are called `coupledValueOld` and `coupledValueOlder`, respectively.
-Similarly,
+This class provides an API for coupling different kinds of variables values into MOOSE systems.
+The following tables summarize the methods it provides.
+
+| Methods for scalar variables | Description |
+| :--- | :--- |
+`coupledValue`^*‡§^ | Value of a coupled variable at q-points
+`coupledGradient`^*‡§^ | Gradient of a coupled variable at q-points
+`coupledSecond`^*‡§^ | Second derivatives of a coupled variable at q-points
+`coupledDot`^†§^ | Time derivative of a coupled variable at q-points
+`coupledDotDot`^†§^ | Second time derivative of a coupled variable at q-points
+`coupledDotDu` | Derivative of the time derivative of a coupled variable at q-points
+`coupledDotDotDu` | Derivative of the second time derivative of a coupled variable at q-points
+`coupledGradientDot`^§^ | Time derivative of the gradient of a coupled variable at q-points
+`coupledGradientDotDot` | Second time derivative of the gradient of a coupled variable at q-points
+
+| Nodal methods for scalar variables | Description |
+| :--- | :--- |
+`coupledNodalValue`^*‡§^ | Value of a coupled variable at nodes
+`coupledNodalDot`^†^ | Time derivative of a coupled variable at nodes
+`coupledNodalDotDot`^†^ | Second time derivative of a coupled variable at nodes
+
+| Methods for vector variables | Description |
+| :--- | :--- |
+`coupledVectorValue`^‡§^ | Value of a coupled vector variable at q-points
+`coupledVectorGradient`^‡§^ | Gradient of a coupled vector variable at q-points
+`coupledCurl`^‡^ | Curl of a coupled vector variable at q-points
+`coupledDiv`^‡^ | Divergence of a coupled vector variable at q-points
```

Comment:
should it be coupledVectorDiv ?
A div of a standard field variable is a thing: dudx + dudy + dudz

Reply:
> A div of a standard field variable is a thing: dudx + dudy + dudz

Do we provide this at all? I'd be surprised if we did. I just tried to follow the same "format" as `coupledCurl`.